app/models/qa/iri_template/url_config.rb

```diff
@@ -1,4 +1,6 @@
 # Provide access to iri template configuration.
+# See https://www.hydra-cg.com/spec/latest/core/#templated-links for information on IRI Templated Links.
+# TODO: It would be good to find a more complete resource describing templated links.
```

Comment:
Added link to documentation of IRI Templated Links to add any devs that come behind me on what it is.